pkg/apis/logging/v1alpha1/fluentd_types.go

```diff
@@ -48,6 +48,7 @@ type FluentdSpec struct {
 type FluentdTLS struct {
 	Enabled    bool   `json:"enabled"`
 	SecretName string `json:"secretName"`
+	SecretType string `json:"secretType"`
```

Comment:
Let this be omitted for bc
```suggestion
	SecretType string `json:"secretType,omitempty"`
```

Reply:
not entirely sure why we need this but I'll add it anyway. Doesn't this apply only when marshalling to JSON?

Reply:
newer kubernetes versions add schema validation which will fail if this is not set in the yaml (which will translate in the background to json of course)